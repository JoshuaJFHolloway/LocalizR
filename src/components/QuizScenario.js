import React from 'react';

const QuizScenario = props => {
  return (
    <h1>{props.scenario}</h1>
  )
};

export default QuizScenario;
