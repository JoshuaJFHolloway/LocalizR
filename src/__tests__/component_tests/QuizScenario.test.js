import React from 'react';
import { shallow } from 'enzyme';
import QuizScenario from '../../components/QuizScenario';

describe('QuizScenario', () => {
  let quizscenario = shallow(<QuizScenario />);

  it('renders correctly', () => {
    expect(quizscenario).toMatchSnapshot();
  })
})
