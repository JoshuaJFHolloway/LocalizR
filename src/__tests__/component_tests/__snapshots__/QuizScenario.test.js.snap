// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuizScenario renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <QuizScenario />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "Scenario_name_here",
    },
    "ref": null,
    "rendered": "Scenario_name_here",
    "type": "h1",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Scenario_name_here",
      },
      "ref": null,
      "rendered": "Scenario_name_here",
      "type": "h1",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
